languages/en.ini

```diff
@@ -858,6 +858,7 @@ Prev = "Prev"
 Preview = "Preview"
 Preview from = "Preview from"
 Previous Title = "Previous Title"
+Primo does not accept an empty query = "Primo does not accept an empty query"
```

Comment:
Might it make sense to add logic to the backend to map the Primo-specific message to something more generic? Would allow reuse of translations and better future-proofing against changes to the message on the Primo side.

Reply:
That's a good idea. I've now updated the translation. Please let me know if you have suggestions for an even better message. I struggled a bit trying to decide on what to call the search backend, but ended up with search target as it's something a user might even understand and separate from e.g. "search service" that could be considered to be VuFind itself.

Reply:
This seems fine to me, though I wonder if it would be better to use a token as the key rather than translating the value into itself -- e.g. empty_search_disallowed -- so if we decide to change the wording in future, we can do so by touching only the language file and not also all instances referencing the string.

Reply:
Makes sense, changed.